packages/react-app/src/components/pages/Bounties/Bounty/index.tsx

```diff
@@ -98,6 +98,10 @@ const BountyDetails = ({
   discordMessageId,
 }: BountyBoardProps): JSX.Element => (
   <Grid gap={6}>
+    <GridItem>
+      <Heading size="sm">HashID</Heading>
+      <Text>{_id}</Text>
```

Comment:
Completely agree! This request came from Iced since hashID is used extensively in discord to help identify the different bounties. So for beta I think it's convenient to just have it available somewhere. I'll keep a note to have this as a "button copy" icon or something